components/tensorboard-controller/config/default/kustomization.yaml

```diff
@@ -6,35 +6,37 @@ namespace: tensorboard-controller-system
 # "wordpress" becomes "alices-wordpress".
 # Note that it should also match with the prefix (text before '-') of the namespace
 # field above.
-namePrefix: tensorboard-controller-
+namePrefix: tensorboard-
```

Comment:
Did you purposefully change this label? I'm getting the following errors when applying the manifests
```
$ kustomize build config/base | k apply -f -
namespace/tensorboard-system created
customresourcedefinition.apiextensions.k8s.io/tensorboards.tensorboard.kubeflow.org created
clusterrole.rbac.authorization.k8s.io/tensorboard-manager-role created
clusterrole.rbac.authorization.k8s.io/tensorboard-metrics-reader created
clusterrole.rbac.authorization.k8s.io/tensorboard-proxy-role created
clusterrolebinding.rbac.authorization.k8s.io/tensorboard-manager-rolebinding created
clusterrolebinding.rbac.authorization.k8s.io/tensorboard-proxy-rolebinding created
configmap/tensorboard-controller-config-mcd64bhg84 created
Error from server (NotFound): error when creating "STDIN": namespaces "tensorboard-controller-system" not found
Error from server (NotFound): error when creating "STDIN": namespaces "tensorboard-controller-system" not found
Error from server (NotFound): error when creating "STDIN": namespaces "tensorboard-controller-system" not found
Error from server (NotFound): error when creating "STDIN": namespaces "tensorboard-controller-system" not found
Error from server (NotFound): error when creating "STDIN": namespaces "system" not found
```

The problem is that the produced Namespace object is named `tensorboard-system` and not `tensorboard-controller-system`. 

Note that because of this rename the `overlays/kubeflow` also fails to be built, since it can't apply the patch that removes the namespace

Reply:
@alembiewski let's change make sure
1. The name of the Deployment will be `tensorboard-controller-deployment`
2. The namespace will be correctly set in `base` as well

I managed to do this by setting the name in the Deployment to `deployment` and removing the namespace everywhere. These are the files I had to modify
```
components/tensorboard-controller/config/base/patches/add_controller_config.yaml
components/tensorboard-controller/config/base/patches/add_service_account.yaml
components/tensorboard-controller/config/default/manager_auth_proxy_patch.yaml
components/tensorboard-controller/config/default/manager_prometheus_metrics_patch.yaml
components/tensorboard-controller/config/default/manager_webhook_patch.yaml
components/tensorboard-controller/config/manager/manager.yaml
```

Can you do this as well and verify the above 2 requirements?

Reply:
@kimwnasptd, I verified that after applying your suggestions the requirements are met.